cmd/ddev/cmd/utils.go

```diff
@@ -46,7 +46,7 @@ func getRequestedProjects(names []string, all bool) ([]*ddevapp.DdevApp, error)
 		if requestedProjectsMap[name], exists = allProjectMap[name]; !exists {
 			p := globalconfig.GetProject(name)
 			if p != nil && p.AppRoot != "" {
-				requestedProjectsMap[name] = &ddevapp.DdevApp{Name: name}
+				requestedProjectsMap[name] = &ddevapp.DdevApp{Name: name, AppRoot: p.AppRoot}
```

Comment:
When you run `ddev start <project>` for a project with missing files, the error does not include the path:

```
Failed to start test-project: ddev can no longer find your project files at . If you would like...
```

And now it does:

```
Failed to start test-project: ddev can no longer find your project files at /home/stas/code/ddev/test-project. If you would like...
```